### PR TITLE
Update to dev12

### PR DESCRIPTION
**Pull request by luca992**
update to dev12
Workaround Table being temporarily removed with AdapterList

### CODE REVIEW

File: app-lifelike/build.gradle [hunk 1]
```diff
@@ -33,21 +33,22 @@ android {
...
-    implementation 'androidx.ui:ui-tooling:0.1.0-dev09'
-    implementation 'androidx.ui:ui-foundation:0.1.0-dev09'
-    implementation 'com.github.zsoltk:compose-router:0.9.0'
+    implementation 'androidx.core:core-ktx:1.3.0'
+    implementation 'androidx.ui:ui-core:0.1.0-dev12'
+    implementation 'androidx.ui:ui-layout:0.1.0-dev12'
+    implementation 'androidx.ui:ui-material:0.1.0-dev12'
+    implementation 'androidx.ui:ui-tooling:0.1.0-dev12'
+    implementation 'androidx.ui:ui-foundation:0.1.0-dev12'
+    implementation project(':router')
```
Comment (zsoltk) [inline app-lifelike/build.gradle]: It's fixed on released version for a reason, exactly to make migration easier.

Sample apps demonstrate Router usage, which doesn't change across Compose dev versions. UI is unimportant (as long as it works). So I'd rather have them use outdated but fixed Compose version than to break them / block upgrading the library itself every time there's a new dev version. 

I'd suggest to revert changes of example apps and only update core.

**Review by zsoltk — CHANGES_REQUESTED** [review]

File: app-lifelike/build.gradle (hunk 1 above)
Comment (luca992) [reply app-lifelike/build.gradle]: Got it. I'll take that out.

Comment (luca992) [reply app-lifelike/build.gradle]: Reverted the samples

**Comment by rachellejane** [comment]
Looks like the requested changes got addressed. Anything else @zsoltk ? Would like to get this update as soon as possible. Can't get 0.11.1 of this to work with Compose dev12.

**Comment by zsoltk** [comment]
@rachellejane thanks for the ping, and sorry about the delay!

**Review by zsoltk — APPROVED** [review]

**Comment by rachellejane** [comment]
Awesome, thank you!

**Pull request merged**